src/config/config.ts

```diff
@@ -292,9 +296,22 @@ export class Config implements IConfig {
   findCommand(id: string, opts?: {must: boolean}): Command.Plugin | undefined
 
   findCommand(id: string, opts: {must?: boolean} = {}): Command.Plugin | undefined {
-    const command = this.commands.find(c => c.id === id || c.aliases.includes(id))
-    if (command) return command
-    if (opts.must) error(`command ${id} not found`)
+    const commands = this.commands.filter(c => c.id === id || c.aliases.includes(id))
+    if (opts.must && commands.length === 0) error(`command ${id} not found`)
+    if (commands.length === 1) return commands[0]
+    // more than one command found across available plugins
+
+    // this.pjson.oclif?.plugins may contain aliases so these need to be resolved against this.pjson.dependencies to get the actual plugin name
+    const resolvedNames = this.resolvePluginAliasNames()
```

Comment:
Plugin entries in the discovered data for loaded plugins carries the NPM name. To account for the possibility of npm alias names, resolve aliases to real npm names